backend/hct_mis_api/apps/program/utils.py

```diff
@@ -264,6 +264,7 @@ def copy_document_per_individual(
             document.individual = individual_representation
             document.program_id = individual_representation.program_id
             document.rdi_merge_status = rdi_merge_status
+            document.status = Document.STATUS_PENDING
```

Comment:
I don't think this is needed, `rdi_merge_status` is passed as PENDING from when calling the `CopyProgramPopulation`

Reply:
Also this needs to stay in an argument of a function, `rdi_merge_status` should be passed here